templates/store/_category-partial.html

```diff
@@ -1,10 +1,16 @@
 <div id="js-snap-{{ category }}">
   <div class="u-fixed-width">
     {% if has_featured %}
+      {% if snaps[0].banner_url %}
+        <a class="p-featured-snap p-featured-snap--banner" href="/{{ snaps[0].package_name }}">
```

Comment:
Does it need `p-featured-snap` class? It seems that `p-featured-snap--banner` styles override most of `p-featured-snap` styles anyway.